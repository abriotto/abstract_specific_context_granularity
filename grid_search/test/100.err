  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  2%|###7                                                                                                                                                                                        | 2/99 [00:00<00:09, 10.14it/s]  4%|#######5                                                                                                                                                                                    | 4/99 [00:00<00:07, 12.35it/s]  6%|###########3                                                                                                                                                                                | 6/99 [00:00<00:08, 11.48it/s]  8%|###############1                                                                                                                                                                            | 8/99 [00:00<00:08, 10.72it/s] 10%|##################8                                                                                                                                                                        | 10/99 [00:01<00:09,  9.07it/s] 12%|######################6                                                                                                                                                                    | 12/99 [00:01<00:09,  9.07it/s] 13%|########################5                                                                                                                                                                  | 13/99 [00:01<00:10,  8.58it/s] 15%|############################3                                                                                                                                                              | 15/99 [00:01<00:09,  9.04it/s] 18%|##################################                                                                                                                                                         | 18/99 [00:01<00:07, 10.60it/s] 20%|#####################################7                                                                                                                                                     | 20/99 [00:02<00:07,  9.91it/s] 21%|#######################################6                                                                                                                                                   | 21/99 [00:02<00:08,  9.04it/s] 23%|###########################################4                                                                                                                                               | 23/99 [00:02<00:08,  8.80it/s] 25%|###############################################2                                                                                                                                           | 25/99 [00:02<00:08,  8.56it/s] 26%|#################################################1                                                                                                                                         | 26/99 [00:02<00:09,  8.04it/s] 28%|####################################################8                                                                                                                                      | 28/99 [00:03<00:08,  8.03it/s] 29%|######################################################7                                                                                                                                    | 29/99 [00:03<00:09,  7.66it/s] 31%|##########################################################5                                                                                                                                | 31/99 [00:03<00:08,  8.06it/s] 32%|############################################################4                                                                                                                              | 32/99 [00:03<00:08,  7.54it/s] 34%|################################################################2                                                                                                                          | 34/99 [00:03<00:08,  7.87it/s] 36%|####################################################################                                                                                                                       | 36/99 [00:04<00:07,  8.95it/s] 38%|#######################################################################7                                                                                                                   | 38/99 [00:04<00:06,  8.94it/s] 40%|###########################################################################5                                                                                                               | 40/99 [00:04<00:05,  9.89it/s] 42%|###############################################################################3                                                                                                           | 42/99 [00:04<00:05, 10.73it/s] 44%|###################################################################################1                                                                                                       | 44/99 [00:04<00:06,  8.77it/s] 46%|######################################################################################8                                                                                                    | 46/99 [00:05<00:06,  8.57it/s] 47%|########################################################################################7                                                                                                  | 47/99 [00:05<00:06,  8.07it/s] 48%|##########################################################################################6                                                                                                | 48/99 [00:05<00:06,  7.30it/s] 51%|##############################################################################################4                                                                                            | 50/99 [00:05<00:06,  7.68it/s] 53%|##################################################################################################2                                                                                        | 52/99 [00:05<00:05,  7.94it/s] 55%|#####################################################################################################9                                                                                     | 54/99 [00:06<00:05,  7.97it/s] 56%|#######################################################################################################8                                                                                   | 55/99 [00:06<00:05,  7.43it/s] 57%|#########################################################################################################7                                                                                 | 56/99 [00:06<00:05,  7.17it/s] 59%|#############################################################################################################5                                                                             | 58/99 [00:06<00:04,  9.38it/s] 61%|#################################################################################################################3                                                                         | 60/99 [00:06<00:04,  7.85it/s] 62%|###################################################################################################################2                                                                       | 61/99 [00:07<00:05,  7.39it/s] 63%|#####################################################################################################################1                                                                     | 62/99 [00:07<00:05,  7.14it/s] 64%|#######################################################################################################################                                                                    | 63/99 [00:07<00:05,  6.54it/s] 65%|########################################################################################################################8                                                                  | 64/99 [00:07<00:05,  6.41it/s] 67%|############################################################################################################################6                                                              | 66/99 [00:07<00:04,  7.08it/s] 68%|##############################################################################################################################5                                                            | 67/99 [00:08<00:04,  6.91it/s] 70%|##################################################################################################################################3                                                        | 69/99 [00:08<00:03,  8.28it/s] 71%|####################################################################################################################################2                                                      | 70/99 [00:08<00:03,  7.74it/s] 73%|########################################################################################################################################                                                   | 72/99 [00:08<00:02,  9.13it/s] 75%|###########################################################################################################################################7                                               | 74/99 [00:08<00:02,  9.08it/s] 76%|#############################################################################################################################################6                                             | 75/99 [00:08<00:02,  8.39it/s] 78%|#################################################################################################################################################4                                         | 77/99 [00:09<00:02,  8.37it/s] 80%|#####################################################################################################################################################2                                     | 79/99 [00:09<00:02,  8.32it/s] 81%|#######################################################################################################################################################1                                   | 80/99 [00:09<00:02,  7.60it/s] 83%|##########################################################################################################################################################8                                | 82/99 [00:09<00:01,  9.67it/s] 85%|##############################################################################################################################################################6                            | 84/99 [00:09<00:01,  9.04it/s] 87%|##################################################################################################################################################################4                        | 86/99 [00:10<00:01,  7.96it/s] 88%|####################################################################################################################################################################3                      | 87/99 [00:10<00:01,  7.46it/s] 90%|########################################################################################################################################################################1                  | 89/99 [00:10<00:01,  9.40it/s] 92%|###########################################################################################################################################################################8               | 91/99 [00:10<00:00,  9.82it/s] 94%|###############################################################################################################################################################################6           | 93/99 [00:10<00:00, 10.62it/s] 96%|###################################################################################################################################################################################4       | 95/99 [00:11<00:00,  9.64it/s] 98%|#######################################################################################################################################################################################2   | 97/99 [00:11<00:00,  9.05it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  9.89it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  8.60it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s]  8%|###############                                                                                                                                                                             | 2/25 [00:00<00:01, 19.03it/s] 16%|##############################                                                                                                                                                              | 4/25 [00:00<00:02,  8.34it/s] 28%|####################################################6                                                                                                                                       | 7/25 [00:00<00:01, 12.75it/s] 36%|###################################################################6                                                                                                                        | 9/25 [00:00<00:01, 14.35it/s] 44%|##################################################################################2                                                                                                        | 11/25 [00:00<00:01, 11.58it/s] 52%|#################################################################################################2                                                                                         | 13/25 [00:01<00:01, 11.73it/s] 60%|################################################################################################################2                                                                          | 15/25 [00:01<00:01,  9.06it/s] 68%|###############################################################################################################################1                                                           | 17/25 [00:01<00:00,  8.73it/s] 76%|##############################################################################################################################################1                                            | 19/25 [00:02<00:00,  7.51it/s] 80%|#####################################################################################################################################################6                                     | 20/25 [00:02<00:00,  7.23it/s] 88%|####################################################################################################################################################################5                      | 22/25 [00:02<00:00,  8.07it/s] 92%|############################################################################################################################################################################               | 23/25 [00:02<00:00,  7.75it/s] 96%|###################################################################################################################################################################################5       | 24/25 [00:02<00:00,  7.17it/s]100%|###########################################################################################################################################################################################| 25/25 [00:02<00:00,  6.97it/s]100%|###########################################################################################################################################################################################| 25/25 [00:02<00:00,  8.76it/s]
