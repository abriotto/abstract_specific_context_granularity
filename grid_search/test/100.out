# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=64", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.99", "--temperature=1.5"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6942542195320129, "acc": 0.5069601535797119, "length": 3.5911953449249268, "mode": "train", "epoch": 1}
{"loss": 0.68295818567276, "acc": 0.600997805595398, "length": 1.7475247383117676, "mode": "test", "epoch": 1}
{"loss": 0.6838503479957581, "acc": 0.5640131831169128, "length": 3.316035032272339, "mode": "train", "epoch": 2}
{"loss": 0.6486755609512329, "acc": 0.7184044122695923, "length": 3.2475247383117676, "mode": "test", "epoch": 2}
{"loss": 0.6051688194274902, "acc": 0.7308276891708374, "length": 3.548492670059204, "mode": "train", "epoch": 3}
{"loss": 0.5472870469093323, "acc": 0.7466447353363037, "length": 3.7838284969329834, "mode": "test", "epoch": 3}
{"loss": 0.505305826663971, "acc": 0.7522740364074707, "length": 3.7500998973846436, "mode": "train", "epoch": 4}
{"loss": 0.5201351046562195, "acc": 0.7563532590866089, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.4798296093940735, "acc": 0.7728462219238281, "length": 3.902859926223755, "mode": "train", "epoch": 5}
{"loss": 0.5054953098297119, "acc": 0.778300404548645, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.46173298358917236, "acc": 0.7916163802146912, "length": 3.962672233581543, "mode": "train", "epoch": 6}
{"loss": 0.4956684708595276, "acc": 0.7822607755661011, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.44330474734306335, "acc": 0.8030272126197815, "length": 3.9777109622955322, "mode": "train", "epoch": 7}
{"loss": 0.4829308092594147, "acc": 0.7873762845993042, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.4279283583164215, "acc": 0.8107538819313049, "length": 3.983287811279297, "mode": "train", "epoch": 8}
{"loss": 0.4627133309841156, "acc": 0.7996700406074524, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.41364479064941406, "acc": 0.8208127021789551, "length": 3.9889049530029297, "mode": "train", "epoch": 9}
{"loss": 0.4472554624080658, "acc": 0.8073431253433228, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.4006861746311188, "acc": 0.8316618800163269, "length": 3.9907350540161133, "mode": "train", "epoch": 10}
{"loss": 0.44319626688957214, "acc": 0.812788724899292, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.3878936469554901, "acc": 0.8364726305007935, "length": 3.992027521133423, "mode": "train", "epoch": 11}
{"loss": 0.43016085028648376, "acc": 0.8140265941619873, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.3765639364719391, "acc": 0.8422005772590637, "length": 3.9936132431030273, "mode": "train", "epoch": 12}
{"loss": 0.4269973933696747, "acc": 0.8176567554473877, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.3735786974430084, "acc": 0.8438074588775635, "length": 3.9939942359924316, "mode": "train", "epoch": 13}
{"loss": 0.41774821281433105, "acc": 0.8179868459701538, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.3637358248233795, "acc": 0.8465298414230347, "length": 3.9944496154785156, "mode": "train", "epoch": 14}
{"loss": 0.4168454110622406, "acc": 0.819472074508667, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.35951361060142517, "acc": 0.8505837917327881, "length": 3.9962706565856934, "mode": "train", "epoch": 15}
{"loss": 0.41227731108665466, "acc": 0.8175743222236633, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.34827959537506104, "acc": 0.8529115915298462, "length": 3.995590925216675, "mode": "train", "epoch": 16}
{"loss": 0.39760351181030273, "acc": 0.8334157466888428, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.34469884634017944, "acc": 0.8527505397796631, "length": 3.996825695037842, "mode": "train", "epoch": 17}
{"loss": 0.3824211359024048, "acc": 0.8402641415596008, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.3362736105918884, "acc": 0.8576617240905762, "length": 3.9971580505371094, "mode": "train", "epoch": 18}
{"loss": 0.3785134255886078, "acc": 0.8455443978309631, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.33137983083724976, "acc": 0.857231855392456, "length": 3.997631788253784, "mode": "train", "epoch": 19}
{"loss": 0.3643936514854431, "acc": 0.8481849431991577, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.31835711002349854, "acc": 0.867626965045929, "length": 3.997201442718506, "mode": "train", "epoch": 20}
{"loss": 0.3576275706291199, "acc": 0.8515677452087402, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.3123445510864258, "acc": 0.8673970699310303, "length": 3.997884511947632, "mode": "train", "epoch": 21}
{"loss": 0.3587460517883301, "acc": 0.8518151044845581, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.31265172362327576, "acc": 0.8681291937828064, "length": 3.9982893466949463, "mode": "train", "epoch": 22}
{"loss": 0.36079201102256775, "acc": 0.8493399620056152, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.3067546784877777, "acc": 0.8723635077476501, "length": 3.9979400634765625, "mode": "train", "epoch": 23}
{"loss": 0.33921799063682556, "acc": 0.8637787699699402, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.2967384457588196, "acc": 0.8790493011474609, "length": 3.997967481613159, "mode": "train", "epoch": 24}
{"loss": 0.33995622396469116, "acc": 0.8646038770675659, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.2948583662509918, "acc": 0.8774443864822388, "length": 3.9983561038970947, "mode": "train", "epoch": 25}
{"loss": 0.33228805661201477, "acc": 0.8636963963508606, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.29368311166763306, "acc": 0.8805090188980103, "length": 3.998831033706665, "mode": "train", "epoch": 26}
{"loss": 0.32748863101005554, "acc": 0.8729372024536133, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.28659892082214355, "acc": 0.8848412036895752, "length": 3.998898506164551, "mode": "train", "epoch": 27}
{"loss": 0.32056623697280884, "acc": 0.8756601214408875, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.28119543194770813, "acc": 0.8841500878334045, "length": 3.998866319656372, "mode": "train", "epoch": 28}
{"loss": 0.32110992074012756, "acc": 0.8760725259780884, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.28129059076309204, "acc": 0.8860417604446411, "length": 3.9987099170684814, "mode": "train", "epoch": 29}
{"loss": 0.3233198821544647, "acc": 0.8707097172737122, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.2766694724559784, "acc": 0.8884156346321106, "length": 3.9985568523406982, "mode": "train", "epoch": 30}
{"loss": 0.31712114810943604, "acc": 0.8792077898979187, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.2759445011615753, "acc": 0.8880478739738464, "length": 3.9992544651031494, "mode": "train", "epoch": 31}
{"loss": 0.32157132029533386, "acc": 0.8777226209640503, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.2741636037826538, "acc": 0.8856506943702698, "length": 3.999406099319458, "mode": "train", "epoch": 32}
{"loss": 0.323940634727478, "acc": 0.8721122145652771, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.2729192078113556, "acc": 0.8892631530761719, "length": 3.9993879795074463, "mode": "train", "epoch": 33}
{"loss": 0.31340491771698, "acc": 0.8710397481918335, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.2718525230884552, "acc": 0.8902847170829773, "length": 3.9994754791259766, "mode": "train", "epoch": 34}
{"loss": 0.31449660658836365, "acc": 0.8818482160568237, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.26688456535339355, "acc": 0.8910574316978455, "length": 3.999509334564209, "mode": "train", "epoch": 35}
{"loss": 0.31406745314598083, "acc": 0.8779703974723816, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.265082448720932, "acc": 0.8924257159233093, "length": 3.999420404434204, "mode": "train", "epoch": 36}
{"loss": 0.3096594512462616, "acc": 0.8784655332565308, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.2629680037498474, "acc": 0.8919366002082825, "length": 3.9994499683380127, "mode": "train", "epoch": 37}
{"loss": 0.3091464936733246, "acc": 0.8772276639938354, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.26066091656684875, "acc": 0.89436936378479, "length": 3.999706983566284, "mode": "train", "epoch": 38}
{"loss": 0.3114822208881378, "acc": 0.8741748929023743, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.26402026414871216, "acc": 0.8908393979072571, "length": 3.999620199203491, "mode": "train", "epoch": 39}
{"loss": 0.304205983877182, "acc": 0.8821781873703003, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.2579185664653778, "acc": 0.8931589722633362, "length": 3.999782085418701, "mode": "train", "epoch": 40}
{"loss": 0.30585089325904846, "acc": 0.8840758800506592, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.2589884400367737, "acc": 0.8943421840667725, "length": 3.9996936321258545, "mode": "train", "epoch": 41}
{"loss": 0.3030077815055847, "acc": 0.8809405565261841, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.25821611285209656, "acc": 0.8943939805030823, "length": 3.999593734741211, "mode": "train", "epoch": 42}
{"loss": 0.30839869379997253, "acc": 0.8815182447433472, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.25749149918556213, "acc": 0.8927131295204163, "length": 3.999584913253784, "mode": "train", "epoch": 43}
{"loss": 0.30781903862953186, "acc": 0.881270706653595, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.25182151794433594, "acc": 0.8960810899734497, "length": 3.99980092048645, "mode": "train", "epoch": 44}
{"loss": 0.2973153293132782, "acc": 0.8858910202980042, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.2523902952671051, "acc": 0.8963236808776855, "length": 3.9996259212493896, "mode": "train", "epoch": 45}
{"loss": 0.3066433072090149, "acc": 0.8759900331497192, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.254401296377182, "acc": 0.895776629447937, "length": 3.9997105598449707, "mode": "train", "epoch": 46}
{"loss": 0.29877740144729614, "acc": 0.8836633563041687, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.2538855969905853, "acc": 0.894208550453186, "length": 3.9997971057891846, "mode": "train", "epoch": 47}
{"loss": 0.3094075620174408, "acc": 0.8784655332565308, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.25543805956840515, "acc": 0.893601655960083, "length": 3.9998621940612793, "mode": "train", "epoch": 48}
{"loss": 0.30104947090148926, "acc": 0.8819308280944824, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.25199535489082336, "acc": 0.8928841948509216, "length": 3.9998040199279785, "mode": "train", "epoch": 49}
{"loss": 0.30058133602142334, "acc": 0.8813533186912537, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.2488614320755005, "acc": 0.8984538316726685, "length": 3.999901056289673, "mode": "train", "epoch": 50}
{"loss": 0.30398598313331604, "acc": 0.87541264295578, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.2509509027004242, "acc": 0.8943747282028198, "length": 3.9999046325683594, "mode": "train", "epoch": 51}
{"loss": 0.302164763212204, "acc": 0.8738449811935425, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.24996429681777954, "acc": 0.8965773582458496, "length": 3.999786853790283, "mode": "train", "epoch": 52}
{"loss": 0.297226220369339, "acc": 0.8839934468269348, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.25425949692726135, "acc": 0.8975428938865662, "length": 3.999833345413208, "mode": "train", "epoch": 53}
{"loss": 0.2980642020702362, "acc": 0.8811882138252258, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.24457857012748718, "acc": 0.8982334733009338, "length": 3.9999213218688965, "mode": "train", "epoch": 54}
{"loss": 0.2926223576068878, "acc": 0.8892740607261658, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.2458907663822174, "acc": 0.8965232968330383, "length": 3.9998364448547363, "mode": "train", "epoch": 55}
{"loss": 0.29283589124679565, "acc": 0.8839934468269348, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.2438018023967743, "acc": 0.8967723250389099, "length": 3.9998698234558105, "mode": "train", "epoch": 56}
{"loss": 0.29562908411026, "acc": 0.8850658535957336, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.24172930419445038, "acc": 0.900244414806366, "length": 3.9999043941497803, "mode": "train", "epoch": 57}
{"loss": 0.29833465814590454, "acc": 0.8830032348632812, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.2380775362253189, "acc": 0.8997230529785156, "length": 3.9999542236328125, "mode": "train", "epoch": 58}
{"loss": 0.2966950237751007, "acc": 0.8851485848426819, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.2369823157787323, "acc": 0.9018715023994446, "length": 3.999913215637207, "mode": "train", "epoch": 59}
{"loss": 0.2918221652507782, "acc": 0.8787129521369934, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.23735733330249786, "acc": 0.9028106331825256, "length": 3.999946117401123, "mode": "train", "epoch": 60}
{"loss": 0.29469034075737, "acc": 0.8814355731010437, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.24007417261600494, "acc": 0.903138279914856, "length": 3.9998908042907715, "mode": "train", "epoch": 61}
{"loss": 0.30025514960289, "acc": 0.8829208016395569, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.2423127293586731, "acc": 0.9001616835594177, "length": 3.9998719692230225, "mode": "train", "epoch": 62}
{"loss": 0.29509973526000977, "acc": 0.8811057209968567, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.2369948774576187, "acc": 0.9014047384262085, "length": 3.999965190887451, "mode": "train", "epoch": 63}
{"loss": 0.2965714931488037, "acc": 0.8811880946159363, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.23410668969154358, "acc": 0.9006590843200684, "length": 3.99991512298584, "mode": "train", "epoch": 64}
{"loss": 0.2855832278728485, "acc": 0.8915017247200012, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.23093768954277039, "acc": 0.9046751260757446, "length": 3.9996306896209717, "mode": "train", "epoch": 65}
{"loss": 0.2842894196510315, "acc": 0.8878713846206665, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.22807566821575165, "acc": 0.9045746326446533, "length": 3.9999709129333496, "mode": "train", "epoch": 66}
{"loss": 0.285343736410141, "acc": 0.8836632370948792, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.22742250561714172, "acc": 0.9064491987228394, "length": 3.9999773502349854, "mode": "train", "epoch": 67}
{"loss": 0.27775558829307556, "acc": 0.8914192914962769, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.2239990383386612, "acc": 0.907962441444397, "length": 3.9998974800109863, "mode": "train", "epoch": 68}
{"loss": 0.28539615869522095, "acc": 0.8865510821342468, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.22372165322303772, "acc": 0.9067250490188599, "length": 3.999987840652466, "mode": "train", "epoch": 69}
{"loss": 0.2783144414424896, "acc": 0.8972773551940918, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.2247604876756668, "acc": 0.9069728851318359, "length": 3.999976873397827, "mode": "train", "epoch": 70}
{"loss": 0.2836177349090576, "acc": 0.8892739415168762, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.22418633103370667, "acc": 0.9039952158927917, "length": 3.9999451637268066, "mode": "train", "epoch": 71}
{"loss": 0.2793727219104767, "acc": 0.8876237869262695, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.23006810247898102, "acc": 0.9031683206558228, "length": 3.999953269958496, "mode": "train", "epoch": 72}
{"loss": 0.2815757393836975, "acc": 0.8867162466049194, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.22189615666866302, "acc": 0.9062833786010742, "length": 3.9999637603759766, "mode": "train", "epoch": 73}
{"loss": 0.2802770733833313, "acc": 0.8818482160568237, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.21547608077526093, "acc": 0.9107764959335327, "length": 3.9999799728393555, "mode": "train", "epoch": 74}
{"loss": 0.271030992269516, "acc": 0.8968647122383118, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.2103835642337799, "acc": 0.9115212559700012, "length": 3.999988079071045, "mode": "train", "epoch": 75}
{"loss": 0.281995952129364, "acc": 0.8909240365028381, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.2158639132976532, "acc": 0.9147453904151917, "length": 3.999976634979248, "mode": "train", "epoch": 76}
{"loss": 0.2858397662639618, "acc": 0.8937294483184814, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.2068125605583191, "acc": 0.9151596426963806, "length": 3.9999945163726807, "mode": "train", "epoch": 77}
{"loss": 0.26803335547447205, "acc": 0.9001649022102356, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.20710454881191254, "acc": 0.9151317477226257, "length": 3.999976634979248, "mode": "train", "epoch": 78}
{"loss": 0.27435705065727234, "acc": 0.8957921266555786, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.20302492380142212, "acc": 0.9173922538757324, "length": 3.9999923706054688, "mode": "train", "epoch": 79}
{"loss": 0.2920411229133606, "acc": 0.8901815414428711, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.23788706958293915, "acc": 0.899503231048584, "length": 3.999976873397827, "mode": "train", "epoch": 80}
{"loss": 0.29080650210380554, "acc": 0.8895214200019836, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.20628176629543304, "acc": 0.9149390459060669, "length": 3.9999923706054688, "mode": "train", "epoch": 81}
{"loss": 0.26505759358406067, "acc": 0.8924092054367065, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.20205222070217133, "acc": 0.9187141060829163, "length": 3.999969005584717, "mode": "train", "epoch": 82}
{"loss": 0.26094505190849304, "acc": 0.8992573618888855, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.19655846059322357, "acc": 0.9190462231636047, "length": 3.99999737739563, "mode": "train", "epoch": 83}
{"loss": 0.2743251621723175, "acc": 0.8901815414428711, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.19915442168712616, "acc": 0.9190679788589478, "length": 3.9998741149902344, "mode": "train", "epoch": 84}
{"loss": 0.26779457926750183, "acc": 0.8966996073722839, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.1965499371290207, "acc": 0.9198992848396301, "length": 3.999976634979248, "mode": "train", "epoch": 85}
{"loss": 0.26304253935813904, "acc": 0.9027228355407715, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.1994209885597229, "acc": 0.9184397459030151, "length": 3.9999966621398926, "mode": "train", "epoch": 86}
{"loss": 0.2591809332370758, "acc": 0.9079208374023438, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.20387190580368042, "acc": 0.9158475399017334, "length": 3.999976634979248, "mode": "train", "epoch": 87}
{"loss": 0.2732029855251312, "acc": 0.900825023651123, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.2022351622581482, "acc": 0.9197075366973877, "length": 3.999995470046997, "mode": "train", "epoch": 88}
{"loss": 0.2787865996360779, "acc": 0.899340033531189, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.20283794403076172, "acc": 0.9200109839439392, "length": 3.9999988079071045, "mode": "train", "epoch": 89}
{"loss": 0.2723526358604431, "acc": 0.8952969908714294, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.2163964956998825, "acc": 0.9130373597145081, "length": 3.9999961853027344, "mode": "train", "epoch": 90}
{"loss": 0.2958442270755768, "acc": 0.8813533186912537, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.2396070659160614, "acc": 0.9019281268119812, "length": 3.9999639987945557, "mode": "train", "epoch": 91}
{"loss": 0.2752029299736023, "acc": 0.8901816606521606, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.2151561975479126, "acc": 0.9134505391120911, "length": 3.9999918937683105, "mode": "train", "epoch": 92}
{"loss": 0.2688296437263489, "acc": 0.9001651406288147, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.20283429324626923, "acc": 0.9156000018119812, "length": 3.9999797344207764, "mode": "train", "epoch": 93}
{"loss": 0.26431673765182495, "acc": 0.9021450281143188, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.19965729117393494, "acc": 0.9192111492156982, "length": 3.9999866485595703, "mode": "train", "epoch": 94}
{"loss": 0.2605184018611908, "acc": 0.9018975496292114, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.19123609364032745, "acc": 0.9231255650520325, "length": 3.9999966621398926, "mode": "train", "epoch": 95}
{"loss": 0.2611706852912903, "acc": 0.9000824093818665, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.18829509615898132, "acc": 0.9248340725898743, "length": 3.9999914169311523, "mode": "train", "epoch": 96}
{"loss": 0.2627197504043579, "acc": 0.9014025330543518, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.1874687373638153, "acc": 0.9262385964393616, "length": 3.9999663829803467, "mode": "train", "epoch": 97}
{"loss": 0.257008820772171, "acc": 0.9069308638572693, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.18500356376171112, "acc": 0.9265159368515015, "length": 3.9999988079071045, "mode": "train", "epoch": 98}
{"loss": 0.24743127822875977, "acc": 0.9067654609680176, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.18550950288772583, "acc": 0.9230429530143738, "length": 3.999998092651367, "mode": "train", "epoch": 99}
{"loss": 0.2562177777290344, "acc": 0.9066005945205688, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.20101530849933624, "acc": 0.9183173179626465, "length": 3.9997916221618652, "mode": "train", "epoch": 100}
{"loss": 0.3225029408931732, "acc": 0.8652640581130981, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.7008619904518127
